lib/mintel/dashboards/components/templates.libsonnet

```diff
@@ -211,4 +211,11 @@ local template = grafana.template;
       },
     ],
   },
-}
+  dashboard_id:: template.new(
```

Comment:
this is a bit confusing as a variable name ... i guess this is dashboard_id as in the omni dashboard_id ? 

can we rename it as such then ?

Also ... this look a very good candidate for an unbount metric cardinality :)

Reply:
I see there is also a "widget_id" , we'll need to keep an eye on those make sure they don't explode

Reply:
I've updated the variable name as you make a valid point.

Regarding the cardinality for dashboard and widget id's, how would we monitor this?

Reply:
not sure yet, we'll figure it out :)